micmac_li3ds/api.py

```diff
@@ -188,14 +189,120 @@ def get_or_create_log(self, typ, obj, parent={}):
         self.log.debug(json.dumps(obj, indent=self.indent))
         return obj
 
-    def get_or_create_transfo(self, transfo, type_, source, target):
+    def get_or_create_sensor(self, name, sensor_type, *, description=None,
```

Comment:
That is the python3 syntax for telling that kargs after the star are not positional.